### DISCUSSION TITLE
New Post Approval Queue

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Revisions#924] [Feature Requests]
Label: Feature Request

As a user, I want to have an queue just like how Revisions Queue but for new page/post

Problem: The default WordPress workflow allow Authors to submit for review, but there is no place for Administrator to see the pending post created by Authors.

Solutions: Introduce New Post creating just like Revisions Queue. Maybe we can combine it both for revisions and new post in 1 dashboard queue.

**Comment by rizaardiyanto1412** [comment]
1 more request

**Comment by stevejburge** [comment]
I talked with @agapetry and he thinks this is a positive idea too

**Comment by klagreca2** [comment]
Wells College is happy to help sponsor this feature, if that helps fasttrack this.

**Comment by stevejburge** [comment]
Thanks @klagreca2. We appreciate that offer. Can you send us a support ticket or contact message to discuss this more?

**Comment by bjoernzosel** [comment]
definitely needed feature!

**Comment by goodermarketing** [comment]
Hoping this feature can be added soon
